### PR TITLE
Removed duplicated split function

### PR DESCRIPTION
**Pull request by ahcorde**
Removed a duplicated version of the split function. Using the one defined in `split.hpp`.

https://github.com/ros2/rcpputils/blob/90adaa69fdd92dacb741e81e6d52df6cbedba05b/include/rcpputils/split.hpp#L89-L95

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: src/find_library.cpp [hunk 1]
```diff
@@ -21,10 +21,13 @@
 #include <fstream>
 #include <sstream>
 #include <string>
+#include <vector>
```
Comment (wjwwood) [inline src/find_library.cpp:24]: Do you need to remove `#include <list>` which is above this line?

Comment (ahcorde) [reply src/find_library.cpp:23]: removed https://github.com/ros2/rcpputils/pull/54/commits/1b24ab54c1e36b3bc57763cdb4755d1bf4dddb03

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10168/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5724/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8260/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10050/)

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10178/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/5733/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8269/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10061/)

**Comment by ahcorde** [comment]
All platforms warning
 - ament_cmake_export_interfaces

Windows warning:
 - spdlog_vendor warning

ci/ws/build/spdlog_vendor
--

**Pull request merged**